src/DIRAC/Core/Utilities/MySQL.py

```diff
@@ -1042,7 +1042,7 @@ def _createTables(self, tableDict, force=False):
                         # cmdList.append( '`%s` %s' % ( forTable, tableDict[forTable]['Fields'][forKey] )
                         cmdList.append(
                             "FOREIGN KEY ( `%s` ) REFERENCES `%s` ( `%s` )"
-                            " ON DELETE RESTRICT" % (key, forTable, forKey)
+                            " ON DELETE CASCADE" % (key, forTable, forKey)
```

Comment:
I think this should not be the default.

Reply:
Elaborate why. It only affects TaskQueueDB and SandboxMetadataDB